### PR TITLE
DOC: add missing whatsnew + classic style to #5503

### PR DESCRIPTION
**Pull request by tacaswell**
attn @WeatherGod

### CODE REVIEW

File: doc/users/whats_new/ticks_rcparams.rst [hunk 1]
```diff
@@ -3,4 +3,8 @@ New rcParams
 
 The parameters `xtick.top`, `xtick.bottom`, `ytick.left`
 and `ytick.right` were added to control where the ticks
-are drawn.
\ No newline at end of file
+are drawn.
+
+``hist.bins`` to control the default number of bins to use in
+`~matplotlib.axes.Axes.hist`.  This can be an ``int``, an array, or
+``'auto'`` if numpy > 1.11 is installed.
```
Comment (mdboom) [inline doc/users/whats_new/ticks_rcparams.rst]: numpy >= 1.11

File: doc/users/whats_new/ticks_rcparams.rst [hunk 2]
```diff
@@ -3,4 +3,8 @@ New rcParams
 
 The parameters `xtick.top`, `xtick.bottom`, `ytick.left`
 and `ytick.right` were added to control where the ticks
-are drawn.
\ No newline at end of file
+are drawn.
+
+``hist.bins`` to control the default number of bins to use in
```
Comment (WeatherGod) [inline doc/users/whats_new/ticks_rcparams.rst]: There is inconsistency with notation for the rcparams. Here, double backticks are used. Above, single backticks are used.

**Comment by tacaswell** [comment]
@mdboom I think this is cleaned up and ready to go.

**Comment by jenshnielsen** [comment]
Looks good to me

**Pull request merged**